flow/connectors/postgres/qrep.go

```diff
@@ -477,6 +477,11 @@ func (c *PostgresConnector) SyncQRepRecords(
 
 // SetupQRepMetadataTables function for postgres connector
 func (c *PostgresConnector) SetupQRepMetadataTables(config *protos.QRepConfig) error {
+	err := c.createMetadataSchema()
+	if err != nil {
+		return fmt.Errorf("error creating metadata schema: %w", err)
+	}
+
 	createQRepMetadataTableTx, err := c.pool.Begin(c.ctx)
```

Comment:
same for this as well?

Reply:
Updated. Raw table setup remains in a transaction